advanced/list-of-log-names.md

```diff
@@ -57,10 +57,10 @@ This category is for things having to do with messages.
 
 | Name                | Emitted when     |
 | ------------------- | ---------------- |
-| CHANNEL_CLEAN       | Someone cleans all the messages from a channel |
-| MESSAGE_BULK_DELETE | Someone bulk deletes a set of messages |
-| MESSAGE_DELETE      | Anyone deletes a message |
-| MESSAGE_EDIT        | Anyone edits a message |
+| CHANNEL_CLEAN       | A moderator cleans all the messages from a channel (which creates a file with the deleted messages) |
```

Comment:
Use passive voice, parenthesis is unsound

`Messages get cleaned from a channel. Includes a file with a backup of the deleted messages`

And the link to the relevant page on "cleaned"